### PR TITLE
Do not shorten postal codes for GB, related to #43

### PR DESCRIPTION
**Pull request by istanishev**
Let's see what I remember from this ;)

### CODE REVIEW

File: PostalCodes.sln [hunk 1]
```diff
@@ -13,12 +13,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{E55142
 		.nuget\packages.config = .nuget\packages.config
 	EndProjectSection
 EndProject
-Project("{7CF6DF6D-3B04-46F8-A40B-537D21BCA0B4}") = "PostalCodes.Documentation", "src\PostalCodes.Documentation\PostalCodes.Documentation.shfbproj", "{F4BAE2A2-656D-4413-B09E-091406584EA3}"
```
Comment (rnowosielski) [inline PostalCodes.sln]: Removing documentation project?

**Comment by rnowosielski** [comment]
I don't remember how this works, but I would assume that there needs to be some nugget version change, right ? Are these changes backwards compatible?

File: PostalCodes.sln (hunk 1 above)
Comment (istanishev) [reply PostalCodes.sln]: Yeah... I was using VS on Mac and did a "shortcut" to fix the build, will revert this.

**Comment by istanishev** [comment]
@rnowosielski Not sure why I cant reply. I also don't quite remember, I think we had to create release branch that indicates the version. The changes are backward-compatible from API standpoint but not quite logically, I'm fine with v3 :)

**Comment by rnowosielski** [comment]
Hehe, I guess there aren't many people using this, but getting the habit of respecting the semantic versioning is valuable in itself :D

**Pull request merged**